Core/src/main/java/com/paypal/android/core/Http.kt

```diff
@@ -47,6 +48,7 @@ internal class Http(
             }.recover {
                 val status = when (it) {
                     is UnknownHostException -> HttpResponse.STATUS_UNKNOWN_HOST
+                    is IllegalStateException -> HttpResponse.SERVER_ERROR
```

Comment:
Could we add a unit test for this additional case?

Reply:
I realized this is covered on _`it returns serverError when the order api call returns an error body`_, it would be the status number "-3".